### DISCUSSION TITLE
Enable an extension globally?

### DISCUSSION THREAD

**Discussion by tastytea** [FreshRSS/FreshRSS#3760] [Q&A] [answered]
I'd like to enable “xExtension-ImageProxy” globally, is it possible?

The `metadata.json` has a field named `"type"` that is set to `"user"`, can I just change it and it works? What's the magic value? 😊

**Comment by tastytea** [comment, marked as answer]
I tried a few things, `"type": "system"` makes it a system extension and it seems to work. Are there any hidden dangers or is the only difference between a system extension and a user extension this field?

**Reply by Frenzie** [reply]
No, that's pretty much it. User preferences vs things you want globally.

**Reply by math-GH** [reply]
Would it make sense to add this tiny - but useful - information into the documentation or into the extension repository readme.md?

**Reply by Frenzie** [reply]
Imo it'd make more sense to just make it an admin toggle as opposed to a metadata toggle but I don't know the rationale behind the design.

**Reply by tastytea** [reply]
It would help if it was an admin toggle. It would make it easier to keep the extension up to date, since I wouldn't have to deal with local changes.

**Reply by math-GH** [reply]
wrote a Feature Request: https://github.com/FreshRSS/FreshRSS/issues/3762

**Comment by Alkarex** [comment]
See also the following configuration option:

https://github.com/FreshRSS/FreshRSS/blob/92ae9c89a4356abc94f2ec5c96a1b783f933fb18/config-user.default.php#L99
